lib/Petal/Cache/Disk.pm

```diff
@@ -9,6 +9,7 @@ package Petal::Cache::Disk;
 use strict;
 use warnings;
 use File::Spec;
+use File::Temp qw/tempfile/;
```

Comment:
Missing from PREREQ_PM in Makefile.PL

Reply:
It's a core module since 5.6.1[1] and other core modules (Carp, Scalar::Util, ...) used by Petal are also not listed so I don't thing it makes much sense to include it.

If preferred I can add it and would also add the other core modules for consistency.

[1] https://perldoc.perl.org/File/Temp.html#HISTORY